### ISSUE TITLE
Windows Font isn't always working

### ISSUE THREAD

**Issue by PandawanFr** [minbrowser/min#248]
I don't know why, but the error page isn't using the font it's given on Windows.
It looks like [this](https://gyazo.com/81a033789ce069faec905b73dea544e7)...

Btw, it works fine on Mac

**Comment by PandawanFr** [comment]
Apparently it's an electron bug...

**Comment by PalmerAL** [comment]
I think this is because all the error pages <a href="https://github.com/minbrowser/min/blob/master/pages/pagebase.css#L8">use the font ".SFNSDisplay-Thin"</a>, which is only available on OS X. Because this font doesn't exist on Windows, the error pages all use the default font.

The fix for this is to use the font list from base.css:

```
	font-family: '.SFNSText-Regular', 'Helvetica Neue', 'Arial', sans-serif;
```

**Comment by PalmerAL** [comment]
This should be fixed by ec8f7246f4740130dc60ac31bcc4f8f80dabac8c.
